pola/rpc_api/openapi-v1.yaml

```diff
@@ -250,6 +236,20 @@ components:
                     official_url:
                       type: string
                       nullable: true
+                    all_company_brands:
```

Comment:
Możemy też to pole zrobić required? Jeśli nie ma brandów niech zwraca pustą listę.

Reply:
done